NuRadioMC/utilities/medium.py

```diff
@@ -80,6 +84,36 @@ def __init__(self):
             )
 
 
+
+class ARA_2022(medium_base.IceModelSimple):
+    def __init__(self):
+        # define model parameters (SPICE 2015/southpole)
```

Comment:
update doc string with correct reference to the paper.

Reply:
Done.